src/main/java/frc/robot/commands/AutomaticShootCommand.java

```diff
@@ -0,0 +1,32 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.ShooterSubsystem;
+
+public class AutomaticShootCommand extends CommandBase {
+    private final ShooterSubsystem shooterSubsystem;
+    private final double targetVel;
+    private final int ballNum;
+    private int totalShoot = 0;
```

Comment:
You could accomplish the same with 1 variable if you subtracted from ballNum every time you shot and the command was finished when it hit zero. If you do this be sure to name it properly like `ballsLeftToShoot` or something